.travis.yml

```diff
@@ -59,7 +59,8 @@ install:
       make source_archive_fat &&
       tar xf bin/neko-*-${TRAVIS_OS_NAME}*.tar.gz &&
       cd neko-*-${TRAVIS_OS_NAME}* &&
-      export PATH=`pwd`:$PATH;
+      export PATH=`pwd`:$PATH &&
+      export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:`pwd`/lib/neko;
```

Comment:
Does `LD_LIBRARY_PATH` really need to be modified?
With `RELOCATABLE=ON`, I think it can find libneko. Or is libneko being outputted to somewhere else?